beeref/__main__.py

```diff
@@ -29,6 +29,14 @@
 
 logger = logging.getLogger(__name__)
 
+class BeeRefApplication(QtWidgets.QApplication):
+
+    def event(self, event):
+        if event.type() == QtCore.QEvent.Type.FileOpen:
+            self.activeWindow().view.open_from_file(event.file())
```

Comment:
I assume BeeRef has only one opened window, which may be a wrong assumption and this code needs to be modified

Reply:
This code fails when users have the `HelpDialog` or `DebugLogDialog` opened. So, will change it such that it finds the instance of `BeeRefMainWindow`.